webbpsf/jupyter_gui.py

```diff
@@ -36,7 +36,7 @@ def show_notebook_interface_wfi(instrument):
     from matplotlib import pyplot as plt
```

Comment:
jupyter_gui is another candidate file to just drop entirely. I can't remember if we formally deprecated it, but it's never been widely used, is not essential functionality, and I think should not be kept. Separate PR.